tests/AppBundle/Service/RestHandler/Report/DeputyCostsEstimateReportUpdateHandlerTest.php

```diff
@@ -26,8 +26,9 @@ class DeputyCostsEstimateReportUpdateHandlerTest extends TestCase
      */
     public function setUp()
     {
+        $date = new \DateTime('now', new \DateTimeZone('Europe/London'));
         $this->report = $this->getMockBuilder(Report::class)
-            ->setConstructorArgs([new Client, Report::TYPE_102, new \DateTime, new \DateTime])
+            ->setConstructorArgs([new Client, Report::TYPE_102, $date, $date])
```

Comment:
I think you need to refresh your branch with latest commits?

Reply:
I've refreshed with `feature-105` - this was related with failures during test runs due to not explicitly setting a timezone.